### ISSUE TITLE
Accuracy of pre-trained model for Tiered Imagenet Dataset

### ISSUE THREAD

**Issue by mayug** [Sha-Lab/FEAT#77]
Hi,

I am trying to recreate the pretrained resnet12 model for tieredimagenet. Could you please tell me what is the best_val_acc_sim and best_val_acc_dist you got when you pretrained on tiered imagenet?

Thanks and Regards
Mayug

**Comment by Han-Jia** [comment]
Hi, 

For the 1-shot test on all meta-val classes, the  best val acc dist is around 16.62% and the best_val_acc_sim is around 18.86%.
